server/utils/validations/adminValidation.js

```diff
@@ -0,0 +1,19 @@
+const yup = require('yup');
+
+const schema = yup.object().shape({
+  id: yup.string().required('Please Enter Client ID'),
```

Comment:
and in here, we do not want the id in here because MongoDB create an id for all instance as this name "_id"